services/exposurelog/values-base.yaml

```diff
@@ -1,25 +1,27 @@
+# WARNING: this is a "playground" deployment
+# using exposurelog's built-in toy butler registry.
 exposurelog:
   pull_secret: pull-secret
 
-  site_id: bleed
-  # Use the built-in toy registry, so set neither butler_uri_1 nor butler_uri_2
+  site_id: test
+  butler_uri_1: hsc_raw
 
   ingress:
     enabled: true
     annotations: {}
     #   kubernetes.io/ingress.class: nginx
     #   kubernetes.io/tls-acme: "true"
     hosts:
-      - host: bleed.lsst.codes
+      - host: base.lsst.codes
         paths:
           - /exposurelog
     tls: []
     #  - secretName: chart-example-tls
     #    hosts:
     #      - chart-example.local
 
-vault_path: secret/k8s_operator/bleed.lsst.codes/postgres
+vault_path: secret/k8s_operator/base.lsst.codes/postgres
```

Comment:
The correct value for the base ``vault_path`` is ``secret/k8s_operator/base-lsp.lsst.codes/postgres``

Reply:
How about this entry:
```
    hosts:
      - host: base.lsst.codes
```
should that be base-lsp.lsst.codes as well?

Reply:
Looking at the summit file clearly it should be. I'll fix all three instances of base.lsst.codes.